### PR TITLE
kata-deploy: Use "centos" as base image

### PR DESCRIPTION
**Pull request by fidencio**
By using centos as base image we can ensure we're using an up-to-date
image, differently than the centos-systemd one*.

*: centos-systemd image has not been updated in the past 3 years or so,
   and quay.io has found several vulnerabilities in the image:
   https://quay.io/repository/kata-containers/kata-deploy?tag=latest&tab=tags

Fixes: #2303

Signed-off-by: Fabiano Fidêncio <fidencio@redhat.com>

### CODE REVIEW

File: tools/packaging/kata-deploy/Dockerfile [hunk 1]
```diff
@@ -2,16 +2,15 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM centos/systemd
+FROM jrei/systemd-centos
```
Comment (jcvenegas) [inline tools/packaging/kata-deploy/Dockerfile:5]: What if we use  centos base image and run this: https://github.com/CentOS/CentOS-Dockerfiles/blob/master/systemd/centos7/Dockerfile

Comment (fidencio) [reply tools/packaging/kata-deploy/Dockerfile:5]: I'm experimenting with a few things for now, but CentOS images still have the a lot of vulnerabilities (the base CentOS image, this one ...).

I'd prefer going with the fedora-minimal one if I can make quay.io properly understand and scan the image, let's see.

**Pull request closed without merging**